ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -750,13 +750,12 @@ <h3 class="title">
       <ng-container matColumnDef="spIndexOrder">
         <th class="table_header" mat-header-cell *matHeaderCellDef>Column Order</th>
         <td mat-cell *matCellDef="let element">
-          <div *ngIf="isIndexEditMode && element.get('spOrder').value">
+          <div *ngIf="isIndexEditMode && element.get('spColName').value">
```

Comment:
Dropping a column from index in spanner is showing removed from source as well - https://drive.google.com/file/d/1RHiW76ff1qjY88xeV28pLjR_VWAZTiaA/view?usp=sharing&resourcekey=0-nZXyKP9RKdVXwa1lCsku7g

Reply:
Solved.